lib/bulk_data_test_kit/v2.0.0_client/urls.rb

```diff
@@ -4,16 +4,16 @@ module BulkDataTestKit
   module BulkDataV200Client
     RESUME_PASS_ROUTE = '/resume_pass'
     BASE_ROUTE = '/fhir'
-    PATIENT_KICKOFF_ROUTE = "#{BASE_ROUTE}/:type/$export".freeze
-    GROUP_KICKOFF_ROUTE = "#{BASE_ROUTE}/:type/:group_id/$export".freeze
+    PATIENT_KICKOFF_ROUTE = "#{BASE_ROUTE}/Patient/$export".freeze
```

Comment:
Did you run into something that required dropping the `:type` here? I noticed the `request_type` method in the kickoff endpoint had to get a little more complicated to account for this.

Reply:
I think I did this originally because I was trying to add session-specific endpoint support, but it was not finding the correct path. I wasn't sure if having more dynamic path components was causing the issue.  I never got it working before I ran out of time, so I can revert this change to keep the code cleaner.